kubeadm/hack/ci-e2e.sh

```diff
@@ -67,6 +67,32 @@ mkdir -p "$ARTIFACTS/logs/"
 python -u ./hack/heartbeat_account.py >> "$ARTIFACTS/logs/boskos.log" 2>&1 &
 HEART_BEAT_PID=$(echo $!)
 
+if [[ -z "$GOOGLE_APPLICATION_CREDENTIALS" ]]; then
+	cat <<EOF
+$GOOGLE_APPLICATION_CREDENTIALS is not set.
+Please set this to the path of the service account used to run this script.
+EOF
+	exit 2
+else
+	gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
```

Comment:
isn't the similar logic from runner.sh triggering?
https://github.com/kubernetes/test-infra/blob/master/images/bootstrap/runner.sh

i do see this in the recent logs:
> Activated service account credentials for: [pr-kubekins@kubernetes-jenkins-pull.iam.gserviceaccount.com]

Reply:
Removed this since yeah I think it's unnecessary.